RELEASE.md

```diff
@@ -10,11 +10,13 @@ Please make sure you have these tools installed before starting this process:
 
 ## Prepare a new release
 
--   [ ] 1.  Inform the engineering team that you are the release manager for a new release in #engineering in our slack (tag engineers) and get a status from each squad on any changes that may be on the master branch of components that may be dealbreakers for the release.
+-   [ ] 1.  Inform the engineering team that you are the release manager for a new release in #engineering in our slack (tag engineers) and get a status from each squad on any changes that may be on the master branch of components that may be dealbreakers for the release
 
--   [ ] 2.  Clone this project on master branch
+-   [ ] 2.  Confirm with every squad if the release can go-ahead
```

Comment:
Can we provide more details on this can be done?

I've seen that what people normally do is asking on the #engineering Slack channel.